packages/compiler-cli/ngcc/test/execution/cluster/worker_spec.ts

```
@@ -141,7 +141,7 @@ describe('startWorker()', () => {
       } as unknown as Task;
 
       startWorker(mockLogger, createCompileFnSpy);
-      cluster.worker.emit('message', {type: 'process-task', task: mockTask});
+      cluster.worker?.emit('message', {type: 'process-task', task: mockTask});
```

I'd replace al these with `cluster.worker!` as crashing the test instead of silently not doing anything seems more desirable.